packages/focusjail/src/FocusJailContainer.spec.js

```diff
@@ -146,13 +149,42 @@ describe('FocusJailContainer', () => {
             <button>Another button</button>
           </>
         );
-        const { container } = render(
+        const { getByTestId } = render(
           <BasicExample focusElem={focusSpy} focusableChildren={focusableChildren} />
         );
 
-        fireEvent.keyDown(container, { keyCode: KEY_CODES.TAB });
+        fireEvent.keyDown(getByTestId('container'), { keyCode: KEY_CODES.TAB });
 
-        expect(focusSpy).toHaveBeenCalledTimes(2);
+        expect(focusSpy).toHaveBeenCalledTimes(3);
+      });
+
+      it('throws error if containerRef is null', () => {
+        const originalError = console.error;
+        let err = null;
+
+        // Jest won't see the error due to it being thrown in an event.
+        // Capture the message here and validate that.
+        window.addEventListener('error', e => {
+          err = e.message;
+        });
```

Comment:
I feel like there should be a better way to do this I did try:

```js
expect(() => {
  const { getByTestId ] = render(...);

  fireEvent.keyDown(getByTestId('container-no-ref'));
}).toThrow();
```

Due to the fact that the error throws in an event listener this expect doesn't receive the thrown error.

Reply:
I use this syntax successfully in react-components. There might be a `console.error` spy leaking somewhere?